app/dashboard/src/layouts/CategorySwitcher.tsx

```diff
@@ -172,16 +170,15 @@ export default function CategorySwitcher(props: CategorySwitcherProps) {
   const { getText } = textProvider.useText()
   const { isOffline } = offlineHooks.useOffline()
   const dispatchAssetEvent = eventListProvider.useDispatchAssetEvent()
-  const setPage = useSetPage()
-  const [, setSearchParams] = useSearchParams()
+  const navigate = useNavigate()
 
   const localBackend = backendProvider.useLocalBackend()
   /** The list of *visible* categories. */
   const categoryData = React.useMemo(
     () =>
       CATEGORY_DATA.filter((data) => {
         switch (data.category) {
-          case Category.local: {
+          case 'local': {
```

Comment:
StringlyTypeScript :P

Reply:
yeah. i guess the idea is mostly that enums are kinda unwieldy and don't offer any real benefit